README-DEVELOPER.md

```diff
@@ -36,7 +36,7 @@ For details on installing them, please read the
 [XBB prerequisites](https://xpack.github.io/xbb/prerequisites/) page.
 
 If you already have a functional configuration from a previous run,
-is is recommended to update **xpm**:
+is recommended to update **xpm**:
```

Comment:
Thank you Mark for catching these. 

I think that here it should have been **it is ...**

Reply:
yep, it should be **it is**, fix now